### ISSUE TITLE
(Public) Round statistics page

### ISSUE THREAD

**Issue by tienne-B** [TabbycatDebate/tabbycat#869]
For the more data-interested participants in tournaments, It would be a neat feature to have a statistics page on the current round (or even previous rounds). This page should only reflect non-results information, but there are many figures that could be represented graphically (or numerically like an "important number" box). Also, some of the graphs already exist, and could simply be transplanted.

In addition, using websockets, most (if not all) of these could update automatically when an event happens.

Some ideas (feel free to add/subtract):
- Ballots status area chart (already exists)
- Percentage of ballots checked-in doughnut chart
- Percentage of feedback submissions sent in doughnut chart

**Comment by czlee** [comment]
Sounds good to me. Would I be right in believing that this sounds more like a "tab progress" page than a (more general) "round statistics" page?

**Comment by tienne-B** [comment]
Yes. I think this would also bode well with your idea of having the ballot graph accessible for previous rounds.

**Comment by philipbelesky** [comment]
Yea, tbh I think a per-round page has much more value for admins that for users. Both because not many tab directors would want that level of transparency and not many users would care. Note also that this would have a lot of value if combined with #392.

For ballots-related items the websocket infrastructure exists; it's just a matter of subscribing to the relevant websocket and accepting the data. Creating a similar channel for feedback paths probably wouldn't be a performance concern (but we should test this). It would, however, mean that we could work towards #841

**Comment by czlee** [comment]
Yeah, it's not really that helpful for participants, though it might not go astray for anxious followers eager to keep tabs on how far away the next draw is. Personally, I wouldn't mind projecting an assistant-view version of this page from GA to try and impress on chairs the degree to which they're holding up the next round. But I'd expect opinions to vary.
